Mergin/plugin.py

```diff
@@ -278,6 +279,24 @@ def _have_writing_permissions(self):
         writersnames = info["access"]["writersnames"]
         return username in writersnames
 
+    def _revise_metadata(self, project_full_name, path):
+        """
+        Validates project name and namespace. If those properties in metadata doesnt match `project_full_name`,
+        metada are updated accordingly. Such case can occur after project transfer, when metadata has not been updated yet.
+        :param project_full_name: project's <namespace>/<name>. This argument suppose to be from server
+        :param path: Absolute location of a project
+        :return bool: True if metadata has been updated.
+        """
+        mp = MerginProject(path)
+        meta_full_name = mp.metadata["name"]
```

Comment:
looking into MerginProject class and metadata can raise some exception, should not we handle this?